src/HealthChecks.UI.Client/UIResponseWriter.cs

```diff
@@ -4,18 +4,19 @@
 using System.Threading.Tasks;
 using System.Text.Json;
 using System.Text.Json.Serialization;
+using System.IO;
 
 namespace HealthChecks.UI.Client
 {
     public static class UIResponseWriter
     {
         const string DEFAULT_CONTENT_TYPE = "application/json";
 
-        public static Task WriteHealthCheckUIResponse(HttpContext httpContext, HealthReport report) => WriteHealthCheckUIResponse(httpContext, report, null);
+        public static async Task WriteHealthCheckUIResponse(HttpContext httpContext, HealthReport report) => await WriteHealthCheckUIResponse(httpContext, report, null);
 
-        public static Task WriteHealthCheckUIResponse(HttpContext httpContext, HealthReport report, Action<JsonSerializerOptions> jsonConfigurator)
+        public static async Task WriteHealthCheckUIResponse(HttpContext httpContext, HealthReport report, Action<JsonSerializerOptions> jsonConfigurator)
         {
-            var response = "{}";
+            var emptyResponse = new byte[] { (byte)'{', (byte)'}' };

```

Comment:
You can move this to static field to reduce allications.

Reply:
In fact you just made more allocations.

Reply:
Agree, just replaced the previous string but this can be done static. Replacing